spec/rubocop/cop/bundler/duplicated_gem_spec.rb

```diff
@@ -85,5 +85,21 @@
         expect(cop.highlights).to eq(["gem 'rubocop', path: '/path/to/gem'"])
       end
     end
+
+    context 'and same gems have different version specifications' do
+      let(:source) { <<-END }
+        if RUBY_VERSION < '2.2.5'
+          gem 'beaker', '~> 2.0', require: false
+          gem 'beaker-rspec', '~> 5.0', require: false
+        else
+          gem 'beaker-rspec', require: false
+        end
+      END
```

Comment:
@tejasbubane

I think it should returns error like ruby "syntax error".

But i don't have any good ideas at this point... 💭